ReactNativeTemplate/package.json

```diff
@@ -11,10 +11,11 @@
         "SalesforceMobileSDK-iOS": "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git#dev"
     },
     "dependencies": {
-        "react": "16.0.0",
-        "react-native": "0.50.4",
+        "react": "16.2.0",
```

Comment:
I wonder if we even need these here since react-native-force should pull in anything it needs.

Reply:
It complains: `UNMET PEER DEPENDENCY`.

Reply:
Hmm that sounds familiar - maybe I did try that before.